sites/www.planetfour.org.conf

```diff
@@ -1,14 +1,28 @@
 server {
     include /etc/nginx/ssl.default.conf;
-    server_name www.planetfour.org;
+    server_name planetfour.org;
 
     location /authors {
-        return 301 https://www.planetfour.org/#/authors;
+        return 301 https://authors.planetfour.org/;
     }
 
     location /results {
-        return 301 https://www.planetfour.org/#/results;
+        return 301 https://www.zooniverse.org/projects/mschwamb/planet-four/about/results;
     }
 
-    include /etc/nginx/proxy.conf;
+
+    location / {
+        return 301 https://www.zooniverse.org/projects/mschwamb/planet-four;
+    }
+}
+
+server {
```

Comment:
I don't think this `server` block is needed, unless I'm misunderstanding something. Without it `authors.planetfour.org` will be served by the [default `server` block](https://github.com/zooniverse/static/blob/be5cd45468139bbf4e11f2247b6bfbbbc4430d38/nginx.conf#L64). (Nginx is set up here so that most domains can be served without needing to be specifically configured, since the [default proxy rule](https://github.com/zooniverse/static/blob/be5cd45468139bbf4e11f2247b6bfbbbc4430d38/nginx-proxy.conf#L7) includes the host name in the S3 URL.)